learning/readme.md

```diff
@@ -1,6 +1,25 @@
-# Learning (Engineering roles)
+# Learning
 
-## Onboarding
+## Learn Tech
+
+Every Friday between 13:30 and 17:00 the team come together to focus on Core Skills.
```

Comment:
The Core Skills mentioned here refers to the old sticker-based learn tech skills.
Could probably do with some re-wording to make it clear that the new tracks are the focus?